src/main/java/com/yevhenberladyniuk/debts/service/DebtService.java

```diff
@@ -0,0 +1,21 @@
+package com.yevhenberladyniuk.debts.service;
+
+import com.yevhenberladyniuk.debts.domain.Debt;
+import com.yevhenberladyniuk.debts.domain.User;
+import com.yevhenberladyniuk.debts.dto.CreateDebt;
+import com.yevhenberladyniuk.debts.dto.DebtDto;
+
+import java.util.List;
+
+public interface DebtService {
+
+    List<Debt> findAllByPartner(Long partnerId, User user);
+
+    void create(CreateDebt createDebt, Long partnerId, User user);
+
+    void deleteById(Long partnerId, Long id, User user);
```

Comment:
Same, no needs of `partnerId`
Better to find `Debt` by id, and get `partnerId` from found `debt`